homeassistant/components/alpha_vantage/sensor.py

```diff
@@ -130,15 +130,18 @@ def update(self):
         _LOGGER.debug("Requesting new data for symbol %s", self._symbol)
         all_values, _ = self._timeseries.get_intraday(self._symbol)
         values = next(iter(all_values.values()))
-        self._attr_native_value = values["1. open"]
+        if "1. open" in values:  # jms
```

Comment:
This will raise an error if `values` is `None`. We check for `None` below. What types do we possibly expect here?

Reply:
You are absolutely right, but values["1. open"] also raises an error if values is None. Normally values is a string or a dictionary. But the dictionary does not always contain the expected items.

Reply:
Ok, so if `values` can be `None` here we should handle that case to avoid an error.

Reply:
fixed